<html><head><title>The Tale of a Neglected Energy Source: Elevated Hydrogen Exposure Affects both Microbial Diversity and Function in Soil &mdash; Supplemental material </title></head><body><div id="content-block"><h1 class="data-supp-article-title">The Tale of a Neglected Energy Source: Elevated Hydrogen Exposure Affects both Microbial Diversity and Function in Soil</h1><div><span class="highwire-journal-article-marker-start"><div class="auto-clean"><span style="font-family: Verdana,Arial,Helvetica,sans-serif;  font-size: 83.33%"><h2>Supplemental material</h2><ul><li><a href="AEM.00275-17_zam999117856s1.pdf">Supplemental file 1</a> - <p>Description of the hypothesized paths constituting the models to predict direct and indirect effects of H<sub>2</sub> exposure on microbial activities (Method S1), dynamic microcosm chamber unit (Method S2), metagenomic analysis pipeline (Method S3), absence of CH<sub>4</sub> production in soil microcosms (Table S1), list of genome bins for which the distribution was influenced by H<sub>2</sub> exposure (Table S2), multiple linear regression equations defining the relative abundances of genome bins outlined in the PCA and the corresponding OTUs (rRNA marker gene amplicon sequencing), according to biotic and abiotic parameters (Table S3), genome bins of potential HOB containing hydrogenase genes based on searches in metagenomic annotation databases (1 gene) and [NiFe]-hydrogenase HMM (7 genes) (Table S4), community-level carbon substrates utilization profiling (Fig. S1), overview of the taxonomic composition of bacterial and fungal communities in soils (Fig. S2), UPGMA agglomerative clustering of soil microcosms according to a Euclidean distance matrix calculated with a Hellinger distance matrix of annotated gene abundance profiles (Fig. S3), and bubble chart representation of the OTUs associated with the 4 genome bins of interest (Fig. S4). </p><p>PDF, 1.7M</p></li><li><a href="AEM.00275-17_zam999117856sd2.xlsx">Supplemental file 2</a> - <p>List of bacterial and fungal OTUs responding to H<sub>2</sub> exposure, bacterial and fungal OTU tables, and raw data (Data Set S1). </p><p>XLSX, 375K</p></li><li><a href="AEM.00275-17_zam999117856sd3.xlsx">Supplemental file 3</a> - <p>Search for genes encoding the large subunit of [NiFe]-hydrogenases using hidden Markov models (Data Set S2). </p><p>XLSX, 42K</p></li><li><a href="AEM.00275-17_zam999117856sd4.xlsx">Supplemental file 4</a> - <p>Gene annotation of genome bins 1, 2, 3, and 6 (Data Set S3).</p><p>XLSX, 1.6M</p></li><li><a href="AEM.00275-17_zam999117856sd5.xlsx">Supplemental file 5</a> - <p>List of oligonucleotides used for PCR-amplicon sequencing (16S rRNA and ITS2) (Data Set S4). </p><p>XLSX, 27K</p></li><li><a href="AEM.00275-17_zam999117856sd6.xlsx">Supplemental file 6</a> - <p>Quality control of metagenomic analysis and contig assembly/genome bin assembly (Data Set S5). </p><p>XLSX, 23K</p></li></ul></span></div><span class="highwire-journal-article-marker-end"> </span></span></div><span id="related-urls"> </span></div></body></html>